tests/system/src/test/java/test/robot/javafx/scene/TabPanePermuteGetTabsTest.java

```diff
@@ -181,6 +189,59 @@ public void testPermuteGetTabsWithMoreTabs() {
         }
     }
 
+    // Test for JDK-8237602
+    @Test
+    public void testPermutGetTabsWithMoreTabs1() {
```

Comment:
Typo: Permut --> Permute

(or, as long as you are changing it anyway, you could rename it now to something other than "permute" since it isn't).

Reply:
Changed the test name to `testAddingNewTabsWithExistingTabsAtSameIndex`. While converting to unit tests, we will have to change the names of other test from this file.